### ISSUE TITLE
How to get the results of -af 'volumedetect'

### ISSUE THREAD

**Issue by RichPC** [protonemedia/laravel-ffmpeg#245]
Hi, Is it possible to use this class for analysing the volume from a segment of an mp3?

With Ffmpeg on the command line I can get the info I need with the command:

`ffmpeg -ss 1 -t 1 -i sample.mp3 -af "volumedetect" -f null /dev/null`

e.g.

```
[Parsed_volumedetect_0 @ 0x563b2e7e2780] n_samples: 88200
[Parsed_volumedetect_0 @ 0x563b2e7e2780] mean_volume: -23.5 dB
[Parsed_volumedetect_0 @ 0x563b2e7e2780] max_volume: -4.5 dB
[Parsed_volumedetect_0 @ 0x563b2e7e2780] histogram_4db: 6
[Parsed_volumedetect_0 @ 0x563b2e7e2780] histogram_5db: 73
[Parsed_volumedetect_0 @ 0x563b2e7e2780] histogram_6db: 76
```

I've tried:

```
   FFMpeg::open('public/audio/sample.mp3')
       ->addFilter(['-ss', 0])
       ->addFilter(['-t', 1])
       ->addFilter(['-af', 'volumedetect'])
       ->export()
       ->inFormat(new \FFMpeg\Format\Audio\Mp3)
       ->save('public/audio/done.mp3');
```

but couldn't find any way to retrieve the volume detection information, plus I don't need to create a new file, I only need the volume information.

I hope someone can help.

**Comment by RichPC** [comment]
I'll manage with shell_exec instead.  For anyone else looking to do this:

```
$command = '/usr/bin/ffmpeg -i /absolute/path/sample.mp3 -af "volumedetect" -f null /dev/null </dev/null 2>&1';
$output = shell_exec($command);
echo $output;
```

**Comment by pascalbaljet** [comment]
I've found a solution how to do this with the package. I'll reopen this issue and post the solution soon (needs some minor changes).

**Comment by pascalbaljet** [comment]
WIP: https://github.com/protonemedia/laravel-ffmpeg/blob/v7-std-listener/tests/VolumeDetectTest.php

**Comment by RichPC** [comment]
nice one @pascalbaljet , thanks for looking at this.

**Comment by pascalbaljet** [comment]
Fixed in v7.5 ([see docs](https://github.com/protonemedia/laravel-ffmpeg/#process-output)).
